pyaedt/misc/toolkit_manager.py

```diff
@@ -0,0 +1,283 @@
+import os
+import tkinter as tk
+from tkinter import ttk
+
+from pyaedt import Desktop
+from pyaedt import is_windows
+from pyaedt.misc.install_extra_toolkits import available_toolkits
+
+env_vars = ["PYAEDT_SCRIPT_VERSION", "PYAEDT_SCRIPT_PORT", "PYAEDT_STUDENT_VERSION"]
+if all(var in os.environ for var in env_vars):
+    version = os.environ["PYAEDT_SCRIPT_VERSION"]
+    version = version[2:6].replace(".", "")
+    port = int(os.environ["PYAEDT_SCRIPT_PORT"])
+    student_version = False if os.environ["PYAEDT_STUDENT_VERSION"] == "False" else True
+else:
+    version = "2024.1"
+    port = 0
+    student_version = False
+
+if is_windows:
+    venv_dir = os.path.join(os.environ["APPDATA"], "pyaedt_env_ide", "toolkits_v{}".format(version))
+    python_exe = os.path.join(venv_dir, "Scripts", "python.exe")
+    package_dir = os.path.join(venv_dir, "Lib", "site-packages")
+
+else:
+    venv_dir = os.path.join(os.environ["HOME"], "pyaedt_env_ide", "toolkits_v{}".format(version))
+    python_exe = os.path.join(venv_dir, "bin", "python")
+    package_dir = os.path.join(venv_dir, "Lib", "site-packages")
+
+
+def create_toolkit_page(frame, open_source_toolkits):
+    """Create page to install toolkit"""
```

Comment:
```suggestion
    """Create page to install toolkit."""
```
I don't really understand this description. Why is a page need to install the toolkit? Should it be something like "Create page to display toolkit on."